src/useCapabilitiesSafe.ts

```diff
@@ -0,0 +1,27 @@
+import { useAccount } from 'wagmi';
+import { useCapabilities } from 'wagmi/experimental';
+import type { UseCapabilitiesSafeParams, WalletCapabilities } from './types';
+
+export function useCapabilitiesSafe({
+  chainId,
+}: UseCapabilitiesSafeParams): WalletCapabilities {
+  const { isConnected } = useAccount();
```

Comment:
Btw as follow up PR, check with @abcrane123 if this will cause a SSR issue.

Reply:
i don't think this will be an issue